examples/step-78/doc/intro.dox

```diff
@@ -0,0 +1,213 @@
+<a name="Intro"></a>
+<h1>Introduction</h1>
+
+\maketitle
+
+\section{Introduction}
+The Black-Scholes equation is a partial differential equation that falls a bit out of the ordinary scheme. It describes what the fair price of a "European call" stock option is. Without going into too much detail, a stock "option" is a contract one can buy from a bank that allows me, but not requires me, to buy a specific stock at a fixed price $K$ at a fixed future time $T$ in the future. The question one would then want to answer as a buyer of such an option is "How much do I think such a contract is worth?", or as the seller "How much do I need to charge for this contract?", both as a function of the time $t<T$ before the contract is up at time $T$ and as a function of the stock price $S$. Fischer Black and Myron Scholes derived a partial differential equation for the fair price $V(S,t)$ for such options under the assumption that stock prices exhibit random price fluctuations with a given level of "volatility" plus a background exponential price increase (which one can think of as the inflation rate that simply devalues all money over time). For their work, Black and Scholes received the Nobel Prize in Economic Sciences in 1997, making this the first tutorial program dealing with a problem for which someone has gotten a Nobel Prize.
```

Comment:
``"European call''

Reply:
Are you asking me to remove the quotes?

Reply:
Latex normally expects different types of quotes (``) at the beginning.

Reply:
Ok, I changed the quotes back and fixed all the align environments. I also added the correct header tags.